include/uapi/sound/sof/abi.h

```diff
@@ -26,7 +26,7 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 15
+#define SOF_ABI_MINOR 16
```

Comment:
The FW change was made with minor version 15

Reply:
> The FW change was made with minor version 15

Isn't minor version 15 for "ipc: cc_version: use fixed length for CC_DESC"? And I think the HDA change is not ready on FW side, right? I am not sure if we can use the same minor version with more than one ipc changes.

Reply:
The FW change for ALH has already been merged and it uses minor version 15. Maybe better to split this change in that case.

Reply:
This can still go in 15. Please update to use 15.

Reply:
> This can still go in 15. Please update to use 15.

Got it. Thanks @lgirdwood